cypress/integration/owm.spec.js

```diff
@@ -4,17 +4,22 @@ describe('OpenWeatherMap layer', function() {
     cy.wait(300)
     cy.window().then((win) => {
       cy.get('.leaflet-tile-pane').children().should('have.length', 1)
-      cy.get('#map-city label').click({ force: true })
+      cy.get('#map-Clouds label').click({ force: true })
       cy.get('.leaflet-tile-pane').children().should('have.length', 2)
-      cy.get('.img')
+      cy.get('img')
```

Comment:
ah, here, it's just going to find ANY image. Let's make it a bit more specific. What is something the clouds layer is DEFINITELY going to display and does it have any characteristic CSS? Does it appear in a named layer?

Reply:
The tiles seem to contain these images

`<img alt="" role="presentation" src="https://c.tile.openweathermap.org/map/clouds/3/5/3.png?appid=4c6704566155a7d0d5d2f107c5156d6e" class="leaflet-tile leaflet-tile-loaded" style="width: 256px; height: 256px; transform: translate3d(1239px, 458px, 0px); opacity: 1;">`

So check if the image src contains the term clouds?